src/views/payment-sheet-views/base-paypal-view.js

```diff
@@ -44,6 +44,7 @@ BasePayPalView.prototype.initialize = function () {
     checkoutJSConfiguration = {
       env: environment,
       style: self.paypalConfiguration.buttonStyle || {},
+      commit: self.paypalConfiguration.commit || false,
```

Comment:
I don't think we need to fall back to false here. If it's not present, it'll just be undefined.